website/src/components/Submission/DataUploadForm.tsx

```diff
@@ -300,7 +301,7 @@ const InnerDataUploadForm = ({
                         <div className='sm:col-span-4 px-8'>
                             {dataUseTermsType === restrictedDataUseTermsOption && (
                                 <p className='block text-sm'>
-                                    Your data will be available on Pathoplexus, under the restricted use terms until{' '}
+                                    Your data will be available on Pathoplexus, under the restricted use terms, until{' '}
```

Comment:
I think this change doesn't make to me. Your change to me implies the data will only be available on Pathoplexus until date X. Whereas it will be available in perpetuity, under the restricted terms until date X.

Reply:
Is this better? Bit more explicit with "also" etc.

> Until the end of the restricted period, your data will be shared on Pathoplexus with restricted use terms.
> Afterwards, the use terms change to open and the data will also be released on the INSDC.

Reply:
I merged the two suggestions - let me know what your thoughts are